Introduce executorch_ops_check

larryliu0820 opened the pull request:
Summary: Introducing `executorch_ops_check`. For any given target, find all the `executorch_generated_lib` targets in the dependent transitive closure and make sure the operators in `executorch_generated_lib` are not overlapping.

Differential Revision: D66560425

lucylq reviewed (approved):
lgtm! note linter failure

larryliu0820 committed:
Introduce executorch_ops_check (#7142)

Summary:

Introducing `executorch_ops_check`. For any given target, find all the `executorch_generated_lib` targets in the dependent transitive closure and make sure the operators in `executorch_generated_lib` are not overlapping.

Reviewed By: lucylq

Differential Revision: D66560425
